v2.md

```diff
@@ -0,0 +1,80 @@
+## Announcements
+### 📢 Release of Gitleaks-Action v2
+_6/13/2022_
+
+<table><tr><td>
+<details>
+<summary><em>Show/hide details</em></summary>
+
+On June 2, 2022, we released [Gitleaks Action v2](https://github.com/gitleaks/gitleaks-action/releases/tag/v2.0.0). There are a boatload of improvements
+in v2, but it also represents a breaking change from the prior version (v1.6.0). We haven't merged v2 to the `master` branch yet because we noticed that
+many users of Gitleaks Action don't pin their version. If you are using `zricethezav/gitleaks-action@master` (or now `gitleaks/gitleaks-action@master`),
+then as soon as we merge v2 to master, your jobs will start failing.
+
+We are planning to complete the merge on **June 20, 2022**. We recommend updating your .yml files to use v2 now so you aren't scrambling to do it after
+your gitleaks-action jobs start failing. As an alternative, you can pin your version to v1.6.0 for now, if you aren't ready to upgrade at the moment.
```

Comment:
It might be time to remove this entirely. Or move it to the bottom. Or move it to a separate Announcements page or something...